R/recipe_commit.R

```diff
@@ -51,9 +51,14 @@ recipe_commit <- function(workdir, config) {
   ## Copy the _files_ over, but we'll roll this back if anything fails
   dest <- copy_report(workdir, dat$name, config)
 
+  DBI::dbBegin(con)
```

Comment:
These changes are unrelated to the PR but I ran into a point where this should have been done - only commit the updates to the DB if all updates work